projects/m1/001-area-of-a-room/solutions/GiorgiaSandionigi/index.js

```diff
@@ -0,0 +1,10 @@
+//inserisco variabili + utilizzo "prompt" per far iserire dati all'utente
+
+let roomWidth = prompt("enter measure Width in meter")
+let roomLenght = prompt("enter measure Lenght in meter")
+let totalArea = roomWidth * roomLenght
+
+
+//utilizzo "alert" per dare messaggio all'utente
+
+alert ( 'the area of the room is ${totalArea} m²')
```

Comment:
You got the exercise logic right and working, the only error here is that the alert at line 10 receives a string instead of a "template literal" (check the backtick char), therefore it will print "${totalArea}" instead of compute the actual result. 
It's not a big issue, just replace:

```js
alert ( 'the area of the room is ${totalArea} m²')
```

with:

```js
alert (`the area of the room is ${totalArea} m²`) // mind the backtick instead of the single quote
```

Well done!!!